modules/core/package.json

```diff
@@ -20,6 +20,8 @@
     "@deck.gl/layers": "^7.3.0",
     "@deck.gl/mesh-layers": "^7.3.0",
     "@deck.gl/react": "^7.3.0",
+    "@emotion/core": "^10.0.22",
```

Comment:
I only see emotion/styled in the **core** module, not `@emotion/core`.

Reply:
`@emotion/styled` needs `@emotion/core`